sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -274,12 +278,23 @@ def get_all_regions_images_df() -> pd.DataFrame:
     return results
 
 
+def fetch_availability_zone_mappings() -> pd.DataFrame:
+    az_mappings = [get_availability_zones.remote(r) for r in ALL_REGIONS]
+    az_mappings = ray.get(az_mappings)
+    az_mappings = pd.concat(az_mappings)
+    return az_mappings
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument(
         '--all-regions',
         action='store_true',
         help='Fetch all global regions, not just the U.S. ones.')
+    parser.add_argument(
+        '--no-az-mappings',
+        action='store_true',
+        help='Not fetching the mapping of availability zone ID to name.')
```

Comment:
```suggestion
        help='Do not fetch the mapping from availability zone IDs to zone names.')
```
typically it's more common to not use double negative. consider --az-mappings?

Reply:
Good point! I made it `az_mapping` and default to `True`.